packages/web/src/pages/sign-up-page/pages/CreateEmailPage/CreateEmailPageDesktop.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Button,
+  ButtonType,
+  Divider,
+  Flex,
+  IconArrowRight,
+  SocialButton,
+  Text
+} from '@audius/harmony'
+import cn from 'classnames'
+
+import audiusLogoColored from 'assets/img/audiusLogoColored.png'
+import { HarmonyTextField } from 'components/form-fields/HarmonyTextField'
+import { Link } from 'components/link'
+import PreloadImage from 'components/preload-image/PreloadImage'
+import { LeftContentContainer } from 'pages/sign-on/components/desktop/LeftContentContainer'
+import {
+  ArtworkContainer,
+  AudiusValues,
+  SignOnContainerDesktop
+} from 'pages/sign-on/components/desktop/PageWithAudiusValues'
+import { SIGN_IN_PAGE } from 'utils/route'
+
+import styles from './CreateEmailPage.module.css'
+import { messages } from './messages'
+
+export const CreateEmailPageDesktop = ({
+  isSubmitting
+}: {
+  isSubmitting: boolean
```

Comment:
useFormikContext the isSubmitting?

Reply:
![](https://media.tenor.com/_GWrgI4QsDQAAAAC/clever-girl-jurassic-park.gif)